### ISSUE TITLE
Unprompted breaks when batch is used

### ISSUE THREAD

**Issue by maxiy01** [ThereforeGames/unprompted#28]
Unprompted breaks on the third (and further) image in the batch.
```
Traceback (most recent call last):
  File "E:\sandbox\stable-diffusion-webui\modules\scripts.py", line 327, in process
    script.process(p, *script_args)
  File "E:\sandbox\stable-diffusion-webui\extensions\unprompted\scripts\unprompted.py", line 88, in process
    p.all_negative_prompts[i] = Unprompted.process_string(Unprompted.shortcode_user_vars["negative_prompt"])
KeyError: 'negative_prompt'
```
I believe that bug is on the line 85 of scripts/unprompted.py:
```
					Unprompted.shortcode_user_vars = {} # < here you empty shortcode_user_var
					Unprompted.shortcode_user_vars["batch_index"] = i
					p.all_prompts[i] = Unprompted.process_string(original_prompt)
					p.all_negative_prompts[i] = Unprompted.process_string(Unprompted.shortcode_user_vars["negative_prompt"]) # < and here you use it
```

**Comment by maxiy01** [comment]
Commenting line 85 seems to fix the issue.

**Comment by ThereforeGames** [comment]
Hi @maxiy01,

This has been fixed in v3.0.0. Thank you for bringing it to my attention!
